installers/conda/win/project.nsi

```diff
@@ -185,16 +186,14 @@ Section "-Core installation"
     # Create shortucts for start menu
     !insertmacro MUI_STARTMENU_WRITE_BEGIN Application
         CreateDirectory "$SMPROGRAMS\$StartMenuFolder"
-        CreateShortCut "$SMPROGRAMS\$StartMenuFolder\${MANTIDWORKBENCH_LINK_NAME}" "$INSTDIR\bin\MantidWorkbench.exe" "" "${WORKBENCH_ICON}"
-        # Mantid Notebook shortcuts are temporarily disabled because they are not working!
-        # CreateShortCut "$SMPROGRAMS\$StartMenuFolder\${MANTIDNOTEBOOK_LINK_NAME}" "cmd.exe" "/C $\"call $INSTDIR\bin\pythonw.exe -m notebook --notebook-dir=%userprofile%$\"" "${NOTEBOOK_ICON}"
+        CreateShortCut "$SMPROGRAMS\$StartMenuFolder\${MANTIDWORKBENCH_LINK_NAME}" "$INSTDIR\bin\MantidWorkbench.exe" "" "$INSTDIR\bin\mantid_workbench.ico"
+        CreateShortCut "$SMPROGRAMS\$StartMenuFolder\${MANTIDNOTEBOOK_LINK_NAME}" "$INSTDIR\bin\python.exe -m jupyter notebook --notebook-dir=%userprofile%" "$INSTDIR\bin\mantid_notebook.ico"
```

Comment:
CreateShortCut requires 4 string arguments. The shortcut is created if the python arguments are moved to a separate string, but unfortunately it doesn't work :(